src/web/detectors/RotationGestureDetector.ts

```diff
@@ -39,16 +39,18 @@ export default class RotationGestureDetector
 
     const [firstPointerID, secondPointerID] = this.keyPointers;
 
-    const firstPointerCoords = tracker.getLastAbsoluteCoords(firstPointerID);
-    const secondPointerCoords = tracker.getLastAbsoluteCoords(secondPointerID);
+    const firstPointerCoords =
+      tracker.getLastViewRelativeCoords(firstPointerID);
+    const secondPointerCoords =
+      tracker.getLastViewRelativeCoords(secondPointerID);
```

Comment:
Huh, does this actually work? In my head, when you try to calculate rotation based on points in the view coordinate space you would get jitter.

When you have two pointers on a view and you rotate them by some angle and then use that angle to rotate the view, that should cancel out. Essentially, the points would be constant in the coordinates of a view, even though they are different in the coordinates of the parent.

Or does `transform` simply not apply to the pointer/touch events sent by the browser?